glib/src/param_spec.rs

```diff
@@ -2065,6 +2065,17 @@ pub trait HasParamSpec {
     fn param_spec_builder() -> Self::BuilderFn;
 }
 
+impl<T: crate::value::ToValueOptional + HasParamSpec> HasParamSpec
```

Comment:
I think this will be obsolete by https://github.com/gtk-rs/gtk-rs-core/pull/967 ?

Reply:
```suggestion
impl<T: HasParamSpec> HasParamSpec
```